x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.tsx

```diff
@@ -13,6 +13,7 @@ import type { UnifiedDataTableProps } from '@kbn/unified-data-table';
 import { UnifiedDataTable, DataLoadingState } from '@kbn/unified-data-table';
 import type { DataView } from '@kbn/data-views-plugin/public';
 import type { EuiDataGridCustomBodyProps, EuiDataGridProps } from '@elastic/eui';
+import { DocumentDetailsRightPanelKey } from '../../../../../flyout/document_details/right';
```

Comment:
```suggestion
import { DocumentDetailsRightPanelKey } from '../../../../../flyout/document_details/shared/constants/panel_keys';
```

I believe the keys were moved recently, so just need to update this and your PR is good to go :)

Reply:
@michaelolo24 .. It is done.